tests/UI/specs/UIIntegration_spec.js

```diff
@@ -57,14 +57,8 @@ describe("UIIntegrationTest", function () { // TODO: Rename to Piwik?
     });
 
     async function screenshotPageWrap() {
-        await page.evaluate(function () {
-          $('#secondNavBar').css('visibility', 'hidden'); // hide navbar so shadow isn't shown on screenshot
```

Comment:
Both `CustomAlerts` and `CustomVariables` have this hack in one place.

_Mentioning it here to keep everything in one place._

Reply:
removed that in the submodule PRs.